test/ConstExtraction/fields.swift

```diff
@@ -71,14 +71,14 @@
 // CHECK-NEXT:        "type": "Swift.Int",
 // CHECK-NEXT:        "isStatic": "false",
 // CHECK-NEXT:        "isComputed": "true",
-// CHECK-NEXT:        "value": "Unknown"
+// CHECK-NEXT:        "value": "3"
```

Comment:
Could we please add a new computed property with a `return` keyword single-statement closure?

Reply:
There were two computed vars (one static, one instance). I converted one to use a return keyword, but I could instead add a new property if that is preferred @artemcm.